src/0x.ts

```diff
@@ -42,6 +43,10 @@ bigNumberConfigs.configure();
  */
 export class ZeroEx {
     /**
+     * Relay constructor. When instantiated - will contain methods to interact with relays using standard relayer API.
```

Comment:
`Relay constructor. Relay is a client for sending requests to Relayers that implement the Standard Relayer API.`